languages/go/exercises/concept/maps/.docs/introduction.md

```diff
@@ -0,0 +1,26 @@
+`map` in go is a key-value type. In other programming language, you might know map as `dict` or associative array. The value of an uninitialized map is `nil`, and you can define map as follows
+
+```go
+  make(map[string]int)
+```
+
+or
+
+```go
+  var foo map[string]int
+  foo = make(map[string]int)
+```
```

Comment:
I don't see the difference between the two ways to create a map, they both use the same function. Is there a difference that I am missing?

Is the other way to define a map:

```go
map[string]int{}
```

?